modules/page-memory/src/test/java/org/apache/ignite/internal/pagememory/TestPageIoModule.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.pagememory;
+
+import java.util.Collection;
+import java.util.List;
+import org.apache.ignite.internal.pagememory.io.IoVersions;
+import org.apache.ignite.internal.pagememory.io.PageIo;
+import org.apache.ignite.internal.pagememory.io.PageIoModule;
+
+/**
+ * Test implementation of {@link PageIoModule}.
+ */
+public class TestPageIoModule implements PageIoModule {
+    /** Last possible value for IO type. */
+    public static final int TEST_PAGE_TYPE = 65535 - 1;
```

Comment:
How about reusing a constant that means 'the max page type'? If it does not exist, it makes sense to create it :)

Reply:
I will create it, it makes sense.